astropy/modeling/core.py

```diff
@@ -3437,5 +3763,132 @@ def _validate_input_shapes(inputs, argnames, n_models, model_set_axis,
     return input_broadcast
 
 
+def remove_axis_from_shape(shape, axis):
```

Comment:
Is this function used anywhere?

Reply:
no, I just like the name. But seriously, I just removed it.